Mathlib/Topology/Homeomorph.lean

```diff
@@ -311,6 +311,14 @@ protected theorem t3Space [T3Space X] (h : X ≃ₜ Y) : T3Space Y :=
 protected theorem denseEmbedding (h : X ≃ₜ Y) : DenseEmbedding h :=
   { h.embedding with dense := h.surjective.denseRange }
 
+universe u v in
+protected lemma totallyDisconnectedSpace {A : Type u} [TopologicalSpace A]
```

Comment:
Why do you need to introduce fresh variables `A` and `B` with their universes? Can't you use the trilion of spaces already in scope?

Reply:
I would try to change it.